include/qpid/dispatch/alloc_pool.h

```diff
@@ -111,7 +111,8 @@ static inline void *qd_alloc_deref_safe_ptr(const qd_alloc_safe_ptr_t *sp)
     void free_##T(T *p); \
     typedef qd_alloc_safe_ptr_t T##_sp; \
     void set_safe_ptr_##T(T *p, T##_sp *sp); \
-    T *safe_deref_##T(T##_sp sp)
+    T *safe_deref_##T(T##_sp sp); \
+    qd_alloc_stats_t *alloc_stats_##T(void)
```

Comment:
It would be nice if we could call this `get_alloc_stats_##T` instead of `alloc_stats_##T`

Reply:
Make this a separate issue.  It was already there, I just added it to the declaration.